hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/schemas/V0530TokenSchema.java

```diff
@@ -41,23 +30,4 @@ public Set<StateDefinition> statesToCreate() {
         return Set.of(StateDefinition.onDisk(
                 AIRDROPS_KEY, PendingAirdropId.PROTOBUF, AccountPendingAirdrop.PROTOBUF, MAX_PENDING_AIRDROPS));
     }
-
-    @Override
-    public void migrate(@NonNull final MigrationContext ctx) {
```

Comment:
There are a couple schema classes in this PR where the migration is removed. Why does removing `KVState.keys()` remove the need to migrate these schemas?

Reply:
The only option we have to not using keys here is iterate over all entities using `netityCounts`. Because we are in `0.60` production code we wont migrate using `0.53` schema. And on `Genesis` this code won't run. So, seems like this code will not be used. 

It is ideal to keep it and use `entityCounts`, but seems too much for some maps like `schedules` and `accounts`